command.go

```diff
@@ -351,9 +407,9 @@ func getCreateOpt(ctx context.Context, args []string, dir string, env map[string
 func (c *Command) getCreateOpts(ctx context.Context) ([]*CreateOptions, error) {
 	out := []*CreateOptions{}
 	catcher := grip.NewBasicCatcher()
-	if c.opts.Hostname != "" {
```

Comment:
I feel like there can be some confusion here due to there being a `(CreateOptions).Hostname` field and a separate `(RemoteCommandOptions).host`. `ApplyFromOpts` could be called with `(CreateOptions).Hostname` set, but if `c.remote.host` is not set to the same value by `(*Command).Host()`, it would unexpectedly ignore `(CreateOptions).Hostname`.

Can we change `(*Command).Host()` to set both `(CreateOptions).Hostname` and `(RemoteComandOptions).host`? Also set `c.remote.host` in `ApplyFromOpts` if `opts.Hostname` is set?

Reply:
I'd be more inclined to remove the hostname from CreateOpts entirely (and move it to process info? just not have it?) it's going to get overridden often by create process, and itsn't used much as is.

Reply:
That also works. I'm fine with removing (CreateOptions).Hostname.